flink-state-backends/flink-statebackend-common/src/main/java/org/apache/flink/state/common/PeriodicMaterializationManager.java

```diff
@@ -251,6 +254,8 @@ private void asyncMaterializationPhase(
                                                 target.handleMaterializationResult(
                                                         snapshotResult, materializationID, upTo);
                                                 metrics.reportCompletedMaterialization();
+                                                metrics.reportMaterializationDuration(
```

Comment:
I think we should report not only the successed materialization but also the failed and one not be called actually due to no state update.
the metric name is "lastMaterializationDuration" not "lastSuccessedMaterializationDuration" or "lastCompletedMaterializationDuration".
WDYT?

Reply:
Thanks @masteryhx , I think it make sense.

Reply:
I think it's a little confusing to mix `successed` and `failed` together.  In fact, the failed materialization duration is meaningless, for me, I prefer to keep `lastMaterializationDuration` and `lastIncSizeOfMaterialization/lastFullSizeOfMaterialization`  consistent.